js/script.js

```diff
@@ -0,0 +1,135 @@
+const playerFactory = (name, symbol) => ({ name, symbol });
+
+const board = (() => {
+  const boardArray = ['', '', '', '', '', '', '', '', ''];
+
+  const checkWin = (currentPlayer) => {
+    let ans = false;
```

Comment:
- [ ] It has a function with the only responsibility of rendering the contents of the gameboard array to the webpage

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/render_requirement.png)

We can have this method in board module as `render`, which will only draw the board from `boardArray` array.

A possible flow would be:
- player clicks on a cell 
- a function updates the `gameboard`, after move validation.
- `board.render()` function fires up to draw the new board